try {
  console.log("try");
  nkm;
} catch (error) {
  console.log("catch");
  console.log(error.message);
}
/* if any error occur in try block catch block will execute */
